Mégane's May updates (#946)
akeneo/pim-helpcenter: content/updates/2021-05/3-New-connect-menu.md

---
pim_announcement_img: ./img/new-connect-entry-menu.png
pim_announcement_alt_img: "New Connect entry menu"
pim_announcement_audience:
- EE
- CE
---

# New Connect entry menu
::: meta-data type="Improvement" features="Productivity" available="mid-May" in="EE,GE"

> In the very new `Connect` menu, you can now find everything related to your connections: the data flow dashboard to monitor the data exchanges on your connections, a direct access to our marketplace to browse all the compatible connectors, and the settings page to manage your connections.

Reviewer: ```suggestion
In the brand new `Connect` menu, you can now find everything related to your connections: the data flow dashboard to monitor data exchanges of your connections, direct access to our marketplace to browse all compatible connectors, and the settings page to manage your connections.
```
